Package incremental 112.35.00: Not a recognised version-control URL

With opam 1.2.0:
```
> opam init
Checking for available remotes: rsync and local, git, darcs.
[NOTE] hg not found, you won't be able to use mercurial repositories unless you install it.
[default] Downloading https://opam.ocaml.org/urls.txt
[default] Downloading https://opam.ocaml.org/index.tar.gz
Updating ~/.opam/repo/compiler-index ...
Updating ~/.opam/compilers/ ...
Updating ~/.opam/repo/package-index ...
Updating ~/.opam/packages/ ...
[ERROR] At ~/.opam/repo/default/packages/incremental/incremental.112.35.00/opam:5:52:
  Not a recognised version-control URL
[ERROR] At ~/.opam/repo/default/packages/incremental/incremental.112.35.00/opam:5:52:
  Not a recognised version-control URL
The following actions will be performed:
 - install   base-unix.base
 - install   base-bigarray.base
 - install   base-threads.base
=== 3 to install ===
...
```

--- a/packages/incremental/incremental.112.35.00/opam
+++ b/packages/incremental/incremental.112.35.00/opam
@@ -2,7 +2,7 @@ opam-version: "1.2"
 maintainer: "opensource@janestreet.com"
 authors: ["Jane Street Group, LLC <opensource@janestreet.com>"]
 homepage: "https://github.com/janestreet/incremental"
-dev-repo: "https://github.com/janestreet/incremental"
+dev-repo: "https://github.com/janestreet/incremental.git"
 bug-reports: "https://github.com/janestreet/incremental/issues"
 license: "Apache-2.0"
 build: [

